### PR TITLE
Cleanup system bindings from Windows when removing a local sync folder

### PR DESCRIPTION
**Pull request by allexzander**
Signed-off-by: alex-z <blackslayer4@gmail.com>

### CODE REVIEW

File: src/common/utility_win.cpp [hunk 1]
```diff
@@ -74,6 +74,40 @@ static void setupFavLink_private(const QString &folder)
         qCWarning(lcUtility) << "linking" << folder << "to" << linkName << "failed!";
 }
 
+static void removeFavLink_private(const QString &folder)
+{
+    const QDir folderDir(QDir::fromNativeSeparators(folder));
```
Comment (mgallien) [inline src/common/utility_win.cpp:79]: is the call to `QDir::fromNativeSeparators` necessary ?
I would have thought that Qt takes care of that by itself

Comment (allexzander) [reply src/common/utility_win.cpp:79]: @mgallien well, yes, it works without it, just took a copy-paste from the function that creates favlink

**Review by mgallien — APPROVED** [review]

**Review by camilasan — APPROVED** [review]

**Pull request merged**